sdk/src/types/api/core.rs

```diff
@@ -57,6 +69,16 @@ impl core::fmt::Display for InfoResponse {
     }
 }
 
+/// Contains the info and the url from the node (useful when multiple nodes are used)
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct NodeInfoResponse {
```

Comment:
I still think this had nothing to do in this file

Reply:
:man_shrugging: maybe we should introduce `core_ext.rs`??

Reply:
I'll clarify: I don't think this has anything to do in the `types` module.
This is created by us for the client -> it should be in the client.
I think it was perfect where it was before.

Reply:
okay, I don't care so much where we put it. To me it personally makes sense to have it very close to the `InfoResponse` type because of the only functional difference.

Reply:
Ok if you don't care then I'm making a decision, let's put it back where it was please

Reply:
okie